src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.UnknownUnix.cs

```diff
@@ -13,8 +13,8 @@ public partial class ProcessThread
         /// </summary>
         private ThreadPriorityLevel PriorityLevelCore
         {
-            get { throw new PlatformNotSupportedException(); }
-            set { throw new PlatformNotSupportedException(); }
+            get { throw new PlatformNotSupportedException(); }  // Why all these?
+            set { throw new PlatformNotSupportedException(); }  // Why all these?
```

Comment:
This UnkownUnix file is only compiled in when building for a new platform, at which point we don't know how to implement these that can't be done on POSIX. We don't ship this.